backend/tests/core/test_i18n.py

```
@@ -132,12 +132,12 @@ def _fake_delay(email, verify_url, language):
         user = User.objects.create_user(
             username="inactive@example.com",
             email="inactive@example.com",
-            password="password123",
+            password="StrongPassword123!",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)